<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APFS</class>
 <widget class="QWidget" name="APFS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Основной стиль для второго окна в стиле 7-Zip - УЛУЧШЕННЫЙ */
QWidget {
    background-color: #f0f0f0;
    color: #000000;
    font-family: &quot;Segoe UI&quot;, Tahoma, sans-serif;
    font-size: 9pt;
}

/* Стиль для QTextEdit - ЯРКИЙ И ВИДИМЫЙ */
QTextEdit {
    background-color: #ffffff;
    border: 2px solid #a0a0a0;
    border-radius: 4px;
    padding: 6px;
    selection-background-color: #0078d4;
    font-family: &quot;Consolas&quot;, &quot;Courier New&quot;, monospace;
    font-size: 10pt;
    color: #000000;
}

QTextEdit:focus {
    border: 2px solid #0078d4;
    background-color: #fffff8;
}

QTextEdit:disabled {
    background-color: #f8f8f8;
    color: #666666;
    border: 2px solid #cccccc;
}

/* Стиль для ProgressBar - ВЫРАЗИТЕЛЬНЫЙ */
QProgressBar {
    background-color: #ffffff;
    border: 2px solid #a0a0a0;
    border-radius: 4px;
    text-align: center;
    color: #000000;
    font-size: 9pt;
    font-weight: bold;
    height: 20px;
}

QProgressBar::chunk {
    background-color: #0078d4;
    border-radius: 3px;
    border: none;
    width: 1px;
}

QProgressBar::chunk:disabled {
    background-color: #a0a0a0;
}

/* Стиль для кнопок - КОНТРАСТНЫЙ */
QPushButton {
    background-color: #e8e8e8;
    border: 2px solid #909090;
    border-radius: 4px;
    padding: 6px 16px;
    font-weight: normal;
    min-width: 80px;
    min-height: 28px;
    color: #000000;
}

QPushButton:hover {
    background-color: #f0f0f0;
    border: 2px solid #707070;
}

QPushButton:pressed {
    background-color: #d8d8d8;
    border: 2px solid #606060;
}

QPushButton:disabled {
    background-color: #f5f5f5;
    color: #a0a0a0;
    border: 2px solid #d0d0d0;
}

/* Кнопка по умолчанию - ЯРКАЯ */
QPushButton[default=&quot;true&quot;] {
    background-color: #0078d4;
    color: white;
    border: 2px solid #005a9e;
    font-weight: bold;
}

QPushButton[default=&quot;true&quot;]:hover {
    background-color: #1084e0;
    border: 2px solid #004578;
}

QPushButton[default=&quot;true&quot;]:pressed {
    background-color: #0060b4;
    border: 2px solid #003966;
}

/* Стиль для ListView - ЧЕТКИЙ */
QListView {
    background-color: #ffffff;
    border: 2px solid #a0a0a0;
    border-radius: 4px;
    outline: 0;
    padding: 2px;
    alternate-background-color: #f8f8f8;
}

QListView::item {
    background-color: transparent;
    border: none;
    padding: 6px 8px;
    margin: 1px;
    height: 22px;
    border-bottom: 1px solid #f0f0f0;
}

QListView::item:hover {
    background-color: #e5f3ff;
    border: none;
}

QListView::item:selected {
    background-color: #0078d4;
    color: white;
    border: none;
}

QListView::item:selected:!active {
    background-color: #e1e1e1;
    color: black;
}

/* Полосы прокрутки - ВИДИМЫЕ */
QScrollBar:vertical {
    background-color: #e8e8e8;
    width: 18px;
    margin: 0px;
    border: 1px solid #a0a0a0;
    border-radius: 2px;
}

QScrollBar::handle:vertical {
    background-color: #a0a0a0;
    border-radius: 6px;
    min-height: 30px;
    margin: 3px;
    border: 1px solid #808080;
}

QScrollBar::handle:vertical:hover {
    background-color: #808080;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: none;
    background: #d0d0d0;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar:horizontal {
    background-color: #e8e8e8;
    height: 18px;
    margin: 0px;
    border: 1px solid #a0a0a0;
    border-radius: 2px;
}

QScrollBar::handle:horizontal {
    background-color: #a0a0a0;
    border-radius: 6px;
    min-width: 30px;
    margin: 3px;
    border: 1px solid #808080;
}

QScrollBar::handle:horizontal:hover {
    background-color: #808080;
}

QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    border: none;
    background: #d0d0d0;
    width: 15px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

/* Метки - ЧЕТКИЕ */
QLabel {
    background-color: transparent;
    color: #000000;
    font-size: 9pt;
    padding: 4px;
    font-weight: normal;
}

QLabel:disabled {
    color: #666666;
}

/* Групповые рамки - ВЫРАЗИТЕЛЬНЫЕ */
QGroupBox {
    border: 2px solid #a0a0a0;
    border-radius: 5px;
    margin-top: 12px;
    padding-top: 18px;
    font-weight: bold;
    background-color: transparent;
    color: #000000;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 12px;
    padding: 0 10px;
    background-color: #f0f0f0;
    color: #000000;
}

/* Линии-разделители */
QFrame[frameShape=&quot;4&quot;] { /* HLine */
    background-color: #a0a0a0;
    color: #a0a0a0;
    max-height: 2px;
}

QFrame[frameShape=&quot;5&quot;] { /* VLine */
    background-color: #a0a0a0;
    color: #a0a0a0;
    max-width: 2px;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>971</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnStart">
      <property name="text">
       <string>Просмотр информации</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnQuit">
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>951</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
